test/files/util.iced

```diff
@@ -0,0 +1,18 @@
+
+{prng} = require 'crypto'
+
+exports.new_uid = new_uid = () -> prng(16).toString('hex')
```

Comment:
Is it ok that the UID doesn't come from a hash of the username? Will we ever try to verify that property on these UIDs?